### ISSUE TITLE
Show incremental label per table

### ISSUE THREAD

**Issue by janmichek** [keboola/kbc-ui#1967]
Uzivatelum by se hodilo videt v detailu tabulky ze je nacitana incrementalne.
Navrhovane reseni - pridat radek Incremental k dalism polozkam.

![image](https://user-images.githubusercontent.com/15363559/44844103-3e71c180-ac4a-11e8-805b-ad57c62abab0.png)

**Comment by janmichek** [comment]
Navrhovane reseni: pridat radek s labelem Incremental / Full load

![image](https://user-images.githubusercontent.com/15363559/44980831-42b31d00-af71-11e8-9319-848684871f13.png)

**Comment by janmichek** [comment]
@Actimel muzes se jeste potom prosim postarat o toto?
Ja si ve Storagi moc nezadevelopim.
Markup a vizual jsem nastrelil nahore jde o to vytahnout si hodnoty  a potom vytvorit podminku

**Comment by Actimel** [comment]
Zkusil jsem dat neco dohromady, ale sekl jsem se na tom ze `table.get('incremental)` imho neni nadefinovany, protoze fraci `false` nehlede na to jaka je tabluka (Vychazim z https://github.com/keboola/kbc-ui/blob/d07689dbfee3c752a5648fb9a0a462dd2031b566/src/scripts/modules/table-browser/react/components/GeneralInfoTab.jsx#L52-L60).

Pote mam pripominku k nastrelu. Nebylo by hezci a i srozumitelnejsi udelat to jako v configuracich tzn. bud fajfka nebo krizek?

<img width="513" alt="snimek obrazovky 2018-09-05 v 23 12 53" src="https://user-images.githubusercontent.com/4041679/45121399-5624da80-b161-11e8-9f83-add380b7fff3.png">

**Comment by janmichek** [comment]
Ano, zatim to neni kdovijak jednotny. Znovupouzivam styl labelu ktery se pouziva v mappingu transformace

**Comment by kacurez** [comment]
@janmichek co pises v https://github.com/keboola/kbc-ui/issues/1967#issue-355508116 tak to sa tyka upravy storage konzole a zdrojaky su tu:
https://github.com/keboola/storage-api-console
To naco ukazuje @Actimel v https://github.com/keboola/kbc-ui/issues/1967#issuecomment-418883628 je table preview komponenta kbc-ui co je light kopia storage konzoli a tam by to mohlo byt tiez.

Technicky zistit ci boli data do tabulky naloadovane incrementalne sa imho da len cez eventy - tj najdes si posledny importny event a zneho a z neho vyzobes hodnotu incremental. Tiez je mozne ze tam taky event nebude a v tom pripade by to malo ukazovat `N/A`. Mozno sa to este uklada k metadatam tabulky ale to myslim ze tam nebude.

**Comment by Halama** [comment]
Ty jo tohle dost zavání tím že to je nějaká feature té tabulky (což by třeba mohla být). Ale pokud se to tam taklhe přidá tak to bude imho hrozně matoucí.

**Comment by Halama** [comment]
Aby to dávalo smysl tak by to měla být informace u toho last importu. 
Něco jako:
![image](https://user-images.githubusercontent.com/903531/45153106-2adad380-b1d3-11e8-85d7-f619f66801f0.png)

Ale i tak to bude potřebovat úpravu na backendu.  Z tech event se to reálně dá vytáhnout jenom v detailu tabulky. Např. v seznamu tabulek už by to byl problém. Takže ta informace o typu posledního loadu musí být uložena v metadatech tabulky.

**Comment by kacurez** [comment]
inac v table preview pod tabom events uz je ta informacia vytiahnuta tak da sa to zistit aj podla toho:
![image](https://user-images.githubusercontent.com/1412120/45154700-ba36b580-b1d8-11e8-9ea9-5219fc98218e.png)
Na zoznam tabuliek by to uz mala byt featura tabulky ako pise Martin.

**Comment by janmichek** [comment]
V jakym je to stavu, pls? @Actimel mame tady nejaky zasek, nebo vsechno potrebny mas?

**Comment by Halama** [comment]
jak jsem psal https://github.com/keboola/kbc-ui/issues/1967#issuecomment-419050508 
Znamená to úpravu na backendu, kterou je potřeba připravit, zařadit a naimplementovat.

**Comment by pivnicek** [comment]
@Halama mame connection issue pro toho? rad bych to udelal

**Comment by Halama** [comment]
@pivnicek nemáme, můžež založit.
